GetIntoTeachingApiTests/Controllers/TeachingEventBuildingsControllerTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using FluentAssertions;
+using GetIntoTeachingApi.Controllers;
+using GetIntoTeachingApi.Models;
+using GetIntoTeachingApi.Services;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace GetIntoTeachingApiTests.Controllers
+{
+    public class TeachingEventBuildingsControllerTests
```

Comment:
Can we add a couple more tests here for the `Authorize` attribute and the caching attributes?

https://github.com/DFE-Digital/get-into-teaching-api/blob/master/GetIntoTeachingApiTests/Controllers/TeachingEventsControllerTests.cs#L49-L71

^ there's actually a typo in that last spec it should read `PrivateShortTermResponseCache_IsPresent()`